config/auth.js

```diff
@@ -52,6 +52,11 @@ export default {
       clientSecret: process.env.GOOGLE_CLIENTSECRET,
       callbackURL: '/auth/google/callback',
       scope: ['https://www.googleapis.com/auth/userinfo.email', 'https://www.googleapis.com/auth/userinfo.profile']
+    },
+    googleExpo: {
```

Comment:
All the settings related to `google` should be under the `google` scope.

Reply:
done